projects/bitfinex.json

```diff
@@ -9,8 +9,8 @@
   "highlight": "A digital asset trading platform offering services for traders and global liquidity providers.",
   "logo": "/img/bitfinex.svg",
   "website": "https://www.bitfinex.com/",
-  "twitter": "",
-  "telegram": "",
+  "twitter": "https://twitter.com/bitfinex?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor",
```

Comment:
we can probably get rid of the query params here. its only used for marketing analytics